portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,67 @@
+html, body {
+  height: 100%;
+  background-color: #F5F5F5;
```

Comment:
Recommend to use only lowercase: https://google.github.io/styleguide/htmlcssguide.html#Capitalization